src/main/java/org/sefglobal/scholarx/config/SecurityConfig.java

```diff
@@ -105,7 +105,7 @@ public CorsConfigurationSource corsConfigurationSource() {
     configuration.setAllowedMethods(ImmutableList.of("GET", "POST", "PUT", "DELETE"));
     configuration.setAllowCredentials(true);
     configuration.setAllowedHeaders(ImmutableList.of("Authorization", "Cache-Control", "Content-Type"));
-    configuration.setAllowedOrigins(ImmutableList.of("http://localhost:3000"));
+    configuration.setAllowedOrigins(ImmutableList.of("https://scholarx.sefglobal.org"));
```

Comment:
@Piumal1999 This will throw the CORS error for local development right? Can't we use an env or something here?

Reply:
Let's add both urls here
```suggestion
    configuration.setAllowedOrigins(ImmutableList.of("http://localhost:3000, "https://scholarx.sefglobal.org"));
```

Reply:
but this will allow anyone to access BE using 3000 right? @Piumal1999

Reply:
Oh right. Then it's good to have an environment variable. 
Or we can just mention this in the README.md

Reply:
yeah let's update the readme then

Reply:
@chirandevinda, As the 6th step of the [Run Locally Section](https://github.com/sef-global/scholarx#run-locally) of README.md file, add the instruction for updating this AllowedOrigins list to contain "http://localhost:3000"

Reply:
Something like this:

6. Update the SecurityConfig.java to allow requests from the origin `http://localhost:3000`.
example:
```
  configuration.setAllowedOrigins(ImmutableList.of("http://localhost:3000"));
```